src/GPL-2.0-only.xml

```diff
@@ -27,11 +27,12 @@
       </p>
     </standardLicenseHeader>
     <notes>
-      This license was released: June 1991. This license identifier refers to when 
-      GPL-2.0-only is being used as opposed to GPL-2.0-or-later. The option to use either that
-      version of the license only or to make it available under any later version of that license
-      is denoted in the standard license header. (The standard license header in the license 
-      itself includes the default "or (at your option) any later version" language.)
+      This license was released: June 1991. This license identifier refers to the choice 
+      to use the code under GPL-2.0-only, as distinguished from use of code under GPL-2.0-or-later
+      (i.e., GPL-2.0 or some later version such as GPL 3, a possible future GPL 4,
+      and so on). The license notice (usually at the head of the file) states which of these
+      applies to the code in the file.
+      <p>The example in the exhibit to the license shows the license notice for the "or later" approach.</p>
```

Comment:
I assume there is no way for FSF to issue/approve an example that does not show the or-later approach?

Reply:
Probably not before we freeze the release for 3.5... which we're hoping to do today / Monday.  =)  But I haven't spoken with them, so maybe @jlovejoy can weigh in here.